Thanks @indigolain seems there are issues with the way you import `ValueType`:

```
/Users/dev/Projects/GitHub/storybook/core/addons/knobs/src/components/types/Options.tsx
  4:27  error  Unable to resolve path to module 'react-select/lib/types'  import/no-unresolved
  4:27  error  Missing file extension for "react-select/lib/types"        import/extensions
```

--- a/.eslintrc.js
+++ b/.eslintrc.js
@@ -35,7 +35,12 @@ module.exports = {
   settings: {
     'import/core-modules': ['enzyme'],
     'import/ignore': ['node_modules\\/(?!@storybook)'],
-    'import/resolver': { node: { extensions: ['.js', '.ts', '.tsx', '.mjs'] } },
+    'import/resolver': {
+      node: {
+        extensions: ['.js', '.ts', '.tsx', '.mjs', '.d.ts'],
+        paths: ['node_modules/', 'node_modules/@types/'],
+      },
+    },
     'html/html-extensions': ['.html'],
   },
   rules: {

--- a/addons/knobs/package.json
+++ b/addons/knobs/package.json
@@ -37,7 +37,7 @@
     "qs": "^6.6.0",
     "react-color": "^2.17.0",
     "react-lifecycles-compat": "^3.0.4",
-    "react-select": "^2.2.0"
+    "react-select": "^3.0.0"
   },
   "peerDependencies": {
     "react": "*"

--- a/examples/dev-kits/stories/addon-decorator.js
+++ b/examples/dev-kits/stories/addon-decorator.js
@@ -1,8 +1,8 @@
 import React from 'react';
 import { storiesOf } from '@storybook/react';
 
-import { withDecorator } from '@storybook/addon-decorator';
 import { withRoundtrip } from '@storybook/addon-roundtrip';
+import { withDecorator } from '@storybook/addon-decorator';
 
 storiesOf('addons|decorator', module)
   .addDecorator(withRoundtrip)

--- a/lib/api/src/version.ts
+++ b/lib/api/src/version.ts
@@ -1,1 +1,1 @@
-export const version = '5.2.0-alpha.40';
+export const version = '5.2.0-alpha.41';

